virtual-hyperscript/svg.js

```diff
@@ -42,6 +44,11 @@ function svg(tagName, properties, children) {
             continue;
         }
 
+        if (key.substr(0, 6) === 'xlink:') {
```

Comment:
@Matt-Esch Thanks for checking it out. I pushed a change to move the namespace into the attribute lookup table and included the `xml:` and `ev:` namespaces as well. Can you take another look, please?